toolbox/linux/why.rst

```diff
@@ -0,0 +1,54 @@
+Why Use Linux?
+==============
+
+Nobody really likes change, and a new operating system means a lot of change.
+Why do we think you should learn it?  Here's a short list:
+
+It's Easier to do Programming
+-----------------------------
+
+Apple makes iOS and OS X development very easy on Macs. Microsoft makes Windows
+development very easy on PCs. However, web development using the most popular
+tools is easiest on Linux, and so is cross platform development. The main reason
+for this is that Linux treats the command line as a "first-class" tool.
+Everything you can do with a GUI [#fn-gui]_ you can also do on the command line,
+and you can usually do even more powerful things on the command line than you
+can with a GUI.
+
+It's a Valuable Skill
+---------------------
+
+Since it's easier to develop many web applications on Linux, it's also a lot
+easier to run them on Linux. Many companies use Linux for their web stack, and
+having basic competency with Linux makes you a much more valuable candidate!
+
+It's Easier to Learn More About
+-------------------------------
+
+Since Linux is open source, you can learn a lot about how your computer works as
+you use it. Once you become comfortable with the basics, you can move to a
+version of Linux that gives you more responsibility for configuring your
+computer (like Arch Linux). This is a great way to learn a lot about the
+practical aspect of how computers work.
```

Comment:
/me has read the kernel source before. Damn wifi drivers and undocumented network functions